corehq/apps/accounting/migrations/0078_auto_20230718_2201.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.20 on 2023-07-18 22:01
```

Comment:
The name of the migration file should be changed to a more meaningful name like `0078_add_web_user_feature_in_other_models`?

Reply:
Yes, changed. Though I'm not sure, should I directly change the file name: ebbc9669166027fd47c39409e77400a9229193de